cypress/utils/utils.ts

```diff
@@ -1383,7 +1383,38 @@ export function enumKeys<O extends object, K extends keyof O = keyof O>(obj: O):
     return Object.keys(obj).filter((k) => Number.isNaN(+k)) as K[];
 }
 
-// This method is patching
+export function isEnabled(selector: string, toBeEnabled?: boolean): void {
+    if (toBeEnabled) {
+        cy.get(selector).should("be.enabled");
+    } else {
+        cy.get(selector).should("not.be.enabled");
+    }
+}
+
+export function isRwxEnabled(): boolean {
+    /**
+     * @remarks
+     * This method is detecting if `rwx_supported` feature is enabled in tackle CR ot not
```

Comment:
typo : or

Reply:
Also , please add that this feature will work from 6.0.1 onwards

Reply:
Fixed